docs/public-networks/how-to/prepare-for-the-merge.md

```diff
@@ -13,7 +13,7 @@ If you're using Besu with [Teku] as the consensus client,
 [prepare Teku for the Merge](https://docs.teku.consensys.net/en/latest/HowTo/Prepare-for-The-Merge/).
 
 You can also
-[test Besu with Teku on the Kiln Merge testnet](../tutorials/merge-testnet.md).
+[run Besu with Teku on the Merge testnet](../tutorials/merge-testnet.md).
```

Comment:
The merge testnet is kiln - so in some places we're saying kiln is deprecated but in other places we're saying use the merge testnet. Probably correct to say it is deprecated but you can still use it leading up to the merge

Reply:
The merge testnet tutorial is being refactored to use the long-term testnets in #1108, so there will be no mention of the merge testnet then.